src/cache/cache.controller.ts

```
@@ -0,0 +1,121 @@
+        ts: '2026-02-25T14:00:00+05:30',
+        params: { status: 'success' },
+        responseCode: 'OK',
+        result: { cleared: true },
+      },
+    },
+  })
+  @ApiBadRequestResponse({ description: 'Bad Request - Invalid tenant ID' })
+  @ApiUnauthorizedResponse({ description: 'Unauthorized' })
+  @ApiInternalServerErrorResponse({ description: 'Internal Server Error' })
+  async clearCache(
+    @Headers('tenantid') tenantId: string,
```

ESLint

[error] Replace `'tenantid'` with `"tenantid"`

(prettier/prettier)